index.html

```diff
@@ -52,7 +52,127 @@ <h1>  I’m Lalla <br>
+                            <li id="detail"><img src="image/Counter.png" alt="Counter"> 2015</li>
+                        </ul>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul>
+                            <li><a href="#" class="programming">HTML</a></li>
+                            <li><a href="#" class="programming">css</a></li>
+                            <li><a href="#" class="programming">javascript</a></li>
+                        </ul>
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+                <div class="job">
+                    <div><img src="image/Snapshoot-Portfolio-3.jpg" alt="job-2 image"></div>
+                    <div class="content">
+                        <h3>Tonic</h3>
+                        <ul>
+                            <li id="campany">Canopy</li>
+                            <li id="detail"><img src="image/Counter.png" alt="Counter"> Back End DEv</li>
+                            <li id="detail"><img src="image/Counter.png" alt="Counter"> 2015</li>
+                        </ul>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul>
+                            <li><a href="#" class="programming">HTML</a></li>
+                            <li><a href="#" class="programming">css</a></li>
+                            <li><a href="#" class="programming">javascript</a></li>
+                        </ul>
+    
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+                <div class="job">
+                    <div><img src="image/SnapshootPortfolio5.jpg" alt="job-2 image"></div>
+                    <div class="content">
+                        <h3>Multi-Post Stories</h3>
+                        <ul>
+                            <li id="campany">Canopy</li>
+                            <li id="detail"><img src="image/Counter.png" alt="Counter"> Back End DEv</li>
+                            <li id="detail"><img src="image/Counter.png" alt="Counter"> 2015</li>
+                        </ul>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul>
+                            <li><a href="#" class="programming">HTML</a></li>
+                            <li><a href="#" class="programming" >css</a></li>
+                            <li><a href="#" class="programming">javascript</a></li>
+                        </ul>
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+            </section>
+            <section class="About" id="About">
```

Comment:
- [ ] Good job implementing the optional suggestion as suggested by the reviewer [here](https://github.com/leloufadel/Portfolio/pull/2#discussion_r1149711220) 🥇🥇 However, because of that, now the background color of project cards is also `#f7f7f9`. But as given in the Figma template, it is `#fff`. See the images below for more clarity 👇


### _Your design before 🔽_
![image](https://user-images.githubusercontent.com/106119105/228225785-e14801a7-414e-49d2-8b82-0f29dbbb47c8.png)

### _Your design now🔽_
![image](https://user-images.githubusercontent.com/106119105/228225544-54bd8ea9-8590-434b-a196-8f3302452c62.png)


### _Figma guidelines 🔽_
![image](https://user-images.githubusercontent.com/106119105/228226299-902bd312-ac7c-46fd-b0a0-49c4b53874cf.png)

**_Please revert back the optional suggestion or apply something like below and make sure that this is working as expected. Please make sure that our task is to follow the Figma design as much as possible. I know you can do that 💪_**
As an example this can be fixed with the following ⚒
```
.job {
  background: white;
  margin: 5rem 0 0 0;
  padding: 1rem 0;
  width: 90vw;
  border: 1px solid #DFE1E6;
  border-radius: 16px;
}

.firstJob {
  margin: 0
}

main {
  position: inherit;
  background-color: white;
  border-bottom-left-radius: 100px;
  padding-bottom: 5rem;
}

.grid-container {
  grid-template-rows: 2fr 1fr;
  gap: 4rem;
}

header {
  background-color: white;
}
```